docs/plugin.md

```diff
@@ -94,9 +94,34 @@ Editor.unuse(Plugins.Header);
 const plugins = ['header', 'table', 'my-plugins', 'link', 'clear', 'logger', 'mode-toggle', 'full-screen'];
 <Editor plugins={plugins} />
 ```
+## Demo for NextJS with custom plugin
```

Comment:
How to implement client-side render is depends on which framework is using, it's not a part of this editor.
I think we can provide a demo, but needn't write codes in documents.